src/admin-dhis2/components/AdminDhis2Page/AdminDhis2Page.jsx

```diff
@@ -13,10 +13,82 @@
  * http://www.gnu.org/licenses.  For additional information contact info@OpenLMIS.org. 
  */
 
-import React from 'react';
+import React, { useMemo, useEffect, useState } from 'react';
 
-const AdminDhis2Page = () => (
```

Comment:
Maybe we should rename this component to `AdminDhis2ServersPage` since there will be only server data? @sniedzielski what do you think?

Reply:
That is a good idea. This new name will be better and more precise the purpose of this page.